src/app/organizations/organization/organization.component.ts

```diff
@@ -84,4 +85,8 @@ export class OrganizationComponent implements OnInit {
   organizationStarGazersChange(): void {
     this.organizationStarGazersClicked = !this.organizationStarGazersClicked;
   }
+
+  getTimeAgoMessage(date: Date) {
```

Comment:
Gary comments whenever I do this -- how often is this invoked?  Probably a lot. It's preferable not to invoke methods for display values from your html.

If it's invoked a lot, you may want to either make this a pipe, or perhaps have the OrganizationQuery return it, although I'm not sure if that's a good practice.